src/backend/api/Fusion.Resources.Api/Controllers/AdminController.cs

```diff
@@ -12,22 +16,69 @@ public class AdminController : ResourceControllerBase
     {
 
         private readonly IOrgUnitCache orgUnitCache;
+        private readonly IMediator mediator;
 
-        public AdminController(IOrgUnitCache orgUnitCache)
+        public AdminController(IOrgUnitCache orgUnitCache, IMediator mediator)
         {
 
             this.orgUnitCache = orgUnitCache;
+            this.mediator = mediator;
         }
 
         [HttpGet("admin/cache/org-units")]
 
         public async Task<ActionResult> CleareCache()
```

Comment:
Small typo there :)

Reply:
IMediator mediator, can't see it 🙈 or do you mean it should be mediatr without the o?